team/drp.rst

```diff
@@ -98,14 +97,31 @@ Options include:
 
 - Bouncing your connection through a `host on the Peyton network <http://www.astro.princeton.edu/docs/Hardware>`_ (this is usually the easiest way to go);
 - Making use of the `University's VPN service <https://www.net.princeton.edu/vpn/>`_.
+- Using the Research Computing gateway.
```

Comment:
Given the Research Computing gateway, is there still a reason to bounce through Peyton? If not, it might be an idea to remove that suggestion for simplicity's sake.

Reply:
I left it in, as it may still be the easiest way to go (no 2FA?).